app/models/ems_event.rb

```diff
@@ -338,6 +338,29 @@ def ems_refresh_target
     ext_management_system
   end
 
+  def src_container_replicator
+    entity = container_replicator_id && ContainerReplicator.find(container_replicator_id)
```

Comment:
I think this raises an exception if not found. Better use ``` X.find_by(:id => id)```
Applicable to src_container_group too